src/components/SubpageOuter/SubpageOuter.jsx

```diff
@@ -25,6 +25,7 @@ const Content = styled.main`
  * Wrapper for all pages. Reason for this is the Navbar not being `fixed` due to styling decisions.
  * Since Navbar is not fixed and taking up space, we need a standard way to adjust the layout to prevent overlaps.
  * @constructor
+ * @param props Component props
```

Comment:
You can remove this. ESLint cries about it, but it's just an annoying quirk about things.